src/utils/filteredTodos.tsx

```diff
@@ -0,0 +1,22 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export const FilteredTodos = (todos: Todo[], filterBy: Status): Todo[] => {
+  let filteredTodos = [...todos];
```

Comment:
You don't need let in this case. You can use const and return operations with an array.